bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdampUP_NNPDF31_NNLO_had.input

```diff
@@ -0,0 +1,90 @@
+! TTbar production parameters
+!randomseed 352345 ! uncomment to set the random seed to a value of your choice.
+                   ! It generates the call RM48IN(352345,0,0) (see the RM48 manual).
+                   ! THIS MAY ONLY AFFECTS THE GENERATION OF POWHEG EVENTS!
+                   ! If POWHEG is interfaced to a shower MC, refer to the shower MC
+                   ! documentation to set its seed.
+
+
+
+numevts NEVENTS
+iseed SEED
+ih1   1        ! hadron 1
+ih2   1        ! hadron 2
+
+
+! To be set only if using LHA pdfs
+lhans1 306000
+lhans2 306000
+! NNPDF31_nnlo_as_0118
+
+
+ebeam1 6500    ! energy of beam 1
+ebeam2 6500    ! energy of beam 2
+qmass  172.5     ! mass of heavy quark in GeV
+facscfact 1    ! factorization scale factor: mufact=muref*facscfact 
+renscfact 1    ! renormalization scale factor: muren=muref*renscfact 
+
+hdamp 397.6125 ! 2.305 x mtop
+
+topdecaymode 00022   ! an integer of 5 digits that are either 0, or 2, representing in 
+                     ! the order the maximum number of the following particles(antiparticles)
+                     ! in the final state: e  mu tau up charm
+                     ! For example
+                     ! 22222    All decays (up to 2 units of everything)
+                     ! 20000    both top go into b l nu (with the appropriate signs)
+                     ! 10011    one top goes into electron (or positron), the other into (any) hadrons,
+                     !          or one top goes into charm, the other into up
+                     ! 00022    Fully hadronic
+                     ! 00002    Fully hadronic with two charms
+                     ! 00011    Fully hadronic with a single charm
+                     ! 00012    Fully hadronic with at least one charm
+
+!semileptonic 1      ! uncomment if you want to filter out only semileptonic events. For example,
```

Comment:
I wonder if is better to remove also this commented commands in the had and dilep, instead of leaving them commented out